Makefile

```diff
@@ -182,6 +188,8 @@ RELEASE_NOTES_VER := $(CAPI_HACK_TOOLS_VER)
 RELEASE_NOTES_BIN := release-notes
 RELEASE_NOTES := $(abspath $(TOOLS_BIN_DIR)/$(RELEASE_NOTES_BIN)-$(RELEASE_NOTES_VER))
 RELEASE_NOTES_PKG := sigs.k8s.io/cluster-api/hack/tools/release
+# let go resolve the tools version via the CAPI version.
+RELEASE_NOTES_MOD_REQUIRE := sigs.k8s.io/cluster-api/hack/tools@$(RELEASE_NOTES_VER)
```

Comment:
```suggestion
RELEASE_NOTES_GO_MOD_REQUIRE := sigs.k8s.io/cluster-api/hack/tools@$(RELEASE_NOTES_VER)
```
nice one!

Reply:
Hm. Should this mean we can drop something as we now have this?

Reply:
This is there so we can re-use the script for non sigs.k8s.io/hack/* binaries.

Reply:
We didn't need it before because it was hardcoded: https://github.com/kubernetes-sigs/cluster-api-provider-vsphere/pull/2310/files#diff-664d5bc044a3e1f516e637b4f9310cddd237799b6a304ebc37a9984d16744bfbL56